modules/ROOT/pages/test-recorder-examples.adoc

```diff
@@ -0,0 +1,66 @@
+= Recording a Test From an Exchange Example
```

Comment:
@kip-mulesoft and @pentreathm 

This is a first draft of an example usage of the Test Recorder.